### PR TITLE
Rename duplicate modules

### LINKED ISSUE #578: Refactor the source code to avoid duplicate naming included modules
shodan.py should be renamed to avoid conflict with the Shodan imported module.

### PR DESCRIPTION
**Pull request by aeferguson-23**
fixes #578
#  #

## 🗣 Description ##

Renaming shodan.py to pe_shodan.py and editing the affected module imports.
Also added a check for fields in API list in pe_shodan.py to eliminate failure upon empty list.

### CODE REVIEW

File: src/pe_source/pe_shodan.py [hunk 1]
```diff
@@ -24,7 +23,7 @@ def run_shodan(self):
 
         # Get orgs from PE database
         pe_orgs = get_orgs()
-
+
```
Comment (jsf9k) [inline src/pe_source/pe_shodan.py:26]:
```suggestion

```

File: src/pe_source/pe_shodan.py [hunk 2]
```diff
@@ -7,13 +7,12 @@
...
 from .data.pe_db.config import shodan_api_init
-from .data.pe_db.db_query import get_orgs
+from .data.pe_db.db_query_source import get_orgs
 from .data.shodan.shodan_search import run_shodan_thread
 
-
 class Shodan:
     """Fetch Shodan data."""
-
+
```
Comment (jsf9k) [inline src/pe_source/pe_shodan.py:15]:
```suggestion

```

**Review by jsf9k — COMMENTED** [review]
Looks good to me, but you added some unnecessary whitespace in a few places.

**Comment by dav3r** [comment]
Please let us know when all of the tests are passing, then I will be happy to review this.

**Pull request closed without merging**